src/OpenTelemetry.Api/Internal/OpenTelemetryApiEventSource.cs

```diff
@@ -108,4 +108,16 @@ public void FailedToInjectBaggage(string format, string error)
     {
         this.WriteEvent(11, format, error);
     }
+
+    [Event(12, Message = "Baggage item key is invalid, key = '{0}'", Level = EventLevel.Warning)]
+    public void BaggageItemKeyIsInvalid(string key)
+    {
+        this.WriteEvent(12, key);
+    }
+
+    [Event(13, Message = "Baggage item value is invalid, value = '{0}'", Level = EventLevel.Warning)]
+    public void BaggageItemValueIsInvalid(string value)
```

Comment:
Q: Should we inform what was the key for the given value? It might be useful for debugging.